xpk/README.md

```diff
@@ -147,6 +147,34 @@ cleanup with a `Cluster Delete`.
     xpk-test --tpu-type=v5litepod-16
     ```
 
+### Workload Priority and Preemption
+* Set the priority level of your workload with `--priority=LEVEL`
+
+  We have five priorities defined: [`verylow`, `low`, `medium`, `high`, `very-high-non-preempt`].
```

Comment:
should we also have a `very-high`? My feeling is that some people have workloads that they don't want to have preempted. So it would be helpful for workloads to be not preemptible (i.e. if high could have that property). It is less helpful for other workloads to not want to preempt.

Reply:
Changed to very-high